app/src/ui/history/commit-list.tsx

```diff
@@ -608,6 +735,19 @@ export class CommitList extends React.Component<ICommitListProps, {}> {
     )
   }
 
+  private canReorder(): boolean {
+    const {
+      onKeyboardReorder: onReorder,
+      disableReordering,
+      isMultiCommitOperationInProgress,
+    } = this.props
+    return (
+      onReorder !== undefined &&
+      disableReordering === false &&
+      isMultiCommitOperationInProgress === false
+    )
+  }
+
```

Comment:
Feels like the destructuring here is counterproductive to readability compared to accessing each prop

```suggestion
  private canReorder = () =>
    this.props.onKeyboardReorder !== undefined &&
    this.props.disableReordering === false &&
    this.props.isMultiCommitOperationInProgress === false

```